Sources/TensorFlow/Layers/Core.swift

```diff
@@ -171,56 +170,25 @@ public struct Dense<Scalar: TensorFlowFloatingPoint>: Layer {
 public extension Dense {
     /// Creates a `Dense` layer with the specified input size, output size, and element-wise
     /// activation function. The weight matrix is created with shape `[inputSize, outputSize]` and
-    /// is initialized using Glorot uniform initialization with the specified generator. The bias
-    /// vector is created with shape `[outputSize]` and is initialized with zeros.
-    ///
-    /// - Parameters:
-    ///   - inputSize: The dimensionality of the input space.
-    ///   - outputSize: The dimensionality of the output space.
-    ///   - activation: The activation function to use. The default value is `identity(_:)`.
-    ///   - generator: The random number generator for initialization.
-    ///
-    /// - Note: Use `init(inputSize:outputSize:activation:seed:)` for faster random initialization.
-    init<G: RandomNumberGenerator>(
-        inputSize: Int,
-        outputSize: Int,
-        activation: @escaping Activation = identity,
-        generator: inout G
-    ) {
-        self.init(weight: Tensor(glorotUniform: [inputSize, outputSize],
-                                 generator: &generator),
-                  bias: Tensor(zeros: [outputSize]),
-                  activation: activation)
-    }
-
-    init(inputSize: Int, outputSize: Int, activation: @escaping Activation = identity) {
-      self.init(inputSize: inputSize, outputSize: outputSize, activation: activation,
-                generator: &PhiloxRandomNumberGenerator.global)
-    }
-}
-
-public extension Dense {
-    /// Creates a `Dense` layer with the specified input size, output size, and element-wise
-    /// activation function. The weight matrix is created with shape `[inputSize, outputSize]` and
-    /// is initialized using Glorot uniform initialization with the specified seed. The bias vector
-    /// is created with shape `[outputSize]` and is initialized with zeros.
+    /// the bias vector is created with shape `[outputSize]`.
     ///
     /// - Parameters:
     ///   - inputSize: The dimensionality of the input space.
     ///   - outputSize: The dimensionality of the output space.
     ///   - activation: The activation function to use. The default value is `identity(_:)`.
-    ///   - seed: The random seed for initialization. The default value is random.
+    ///   - weightsInitializer: Initializer to use for the weight parameters.
+    ///   - biasInitializer: Initializer to use for the bias parameters.
     init(
         inputSize: Int,
         outputSize: Int,
-        activation: @escaping Activation = identity,
-        seed: (Int32, Int32) = (Int32.random(in: Int32.min..<Int32.max),
-                                Int32.random(in: Int32.min..<Int32.max))
+        activation: @escaping Activation<Scalar> = identity,
+        weightsInitializer: ParameterInitializer<Scalar> = glorotUniform(),
```

Comment:
We've been treating the weight tensor as a single object, so it'd be good to keep it consistent.
```suggestion
        weightInitializer: ParameterInitializer<Scalar> = glorotUniform(),
```